Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -438,6 +438,15 @@ int main(int argc, char **argv, char **const envp) {
     else {
       Allocator = FEX::HLE::CreatePassthroughAllocator();
     }
+
+    // Now that the upper 32-bit address space is blocked for future allocations,
+    // exhaust all of jemalloc's remaining internal allocations that it reserved before.
+    // TODO: It's unclear how reliably this exhausts those reserves
+    void* data;
+    do {
+      data = malloc(0x1);
```

Comment:
This fails the glibc allocator CI checks. Is `FEXCore::Allocator::malloc` the right alternative to use here?

Reply:
The fault enabling would need to be temporarily disabled around this, since it is expected that FEX is abusing the glibc allocator functions here.